app/controllers/review_bids_controller.rb

```diff
@@ -3,22 +3,15 @@ class ReviewBidsController < ApplicationController
   require 'uri'
   require 'net/http'
   require 'rest_client'
+  include AuthorizationHelper
 
   # action allowed function checks the action allowed based on the user working
   def action_allowed?
     case params[:action]
     when 'show', 'set_priority', 'index'
-      ['Instructor',
-       'Teaching Assistant',
-       'Administrator',
-       'Super-Administrator',
-       'Student'].include?(current_role_name) &&
-        ((%w[list].include? action_name) ? are_needed_authorizations_present?(params[:id], 'participant', 'reader', 'submitter', 'reviewer') : true)
+      current_user_has_student_privileges? && ((%w[list].include? action_name) ? are_needed_authorizations_present?(params[:id], 'participant', 'reader', 'submitter', 'reviewer') : true)
```

Comment:
This line is hard to follow.  Either break it up into shorter statements that are easier to read, and/or comment on what it is doing.